app/services/news_feed.rb

```diff
@@ -34,6 +34,7 @@ class NewsFeed
   USER_FOLLOWERS_PREFIX         = "user_followers:"
   USER_FOLLOWING_PREFIX         = "user_following:"
   ACTIVE_FOLLOWED_USERS_PREFIX  = "active_followed_users:"
+  ACTIVE_JOINED_GROUPS_PREFIX   = "active_joined_groups:"
```

Comment:
We don't need these keys anymore since we're not using Redis as a cache for this information now.